Ginger/GingerTest/GingerAutomator.cs

```diff
@@ -196,6 +196,8 @@ internal void OpenSolution(string folder)
             GingerPOMBase.Dispatcher.Invoke(() =>
             {
                 // TODO: do it like user with open solution page
+                Console.WriteLine("Error in Automator-OpenSolution");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this logging statement.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=4707297)